### DISCUSSION TITLE
Show size of zips

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#83010] [Repositories]
### Select Topic Area

Bug

### Body

On e.g., https://github.com/zamojski/TowerCollector/releases/tag/2.14.4
we see three downloadable items.

But only one, the .apk, shows how much the user must download.

The two following items, .zip and .tar.gz, are missing their sizes.

The user can only cross their fingers and hope the files aren't too large.

**Comment by kavindyasinthasilva** [comment]
Most GitHub repositories have an "Issues" tab where you can report problems or suggest enhancements.

**Reply by jidanni** [reply]
How could the missing numbers be a fault of the repository and not the GitHub interface?
